### ISSUE TITLE
Process transform shorthand props in a different order

### ISSUE THREAD

**Issue by notoriousb1t** [just-animate/just-animate#22]
What order should the shorthand properties: skew, rotate, translate, and scale be in?

**Comment by notoriousb1t** [comment]
Test codepen here: http://codepen.io/notoriousb1t/pen/EgWgWp?editors=0010

Chrome is supporting shorthand properties for scale, rotate and translate.  May want to emulate that transform: https://drafts.csswg.org/css-transforms-2/#ctm

Maybe instead they should be added to the transform in the order they are on the property:

```javascript
{ translate: '20px', rotate: '20deg', scale: 1.1 }
```
would render differently than


```javascript
{ rotate: '20deg', scale: 1.1, translate: '20px' }
```

**Comment by notoriousb1t** [comment]
Implemented this property order in 90753977516a9dee1cc124b2d6f6f9e0283022e7

- perspective
- matrix
- translate
- translate3d
- translateX
- translateY    
- translateZ
- skew
- skewX
- skewY
- scale    
- scale3d
- scaleX
- scaleY
- scaleZ
- rotate
- rotate3d    
- rotateX
- rotateY
- rotateZ
